handler/dummy/dummy.go

```diff
@@ -0,0 +1,7 @@
+package dummy
+
+type Dummy struct{}
```

Comment:
Adding the dummy implementation here because I needed something that runs, but doesn't do anything. Is often useful if you want to test worker engines/handlers in isolation.